NativeApp/src/components/User/container.js

```diff
@@ -17,21 +28,22 @@ export default Container => class extends Component {
   }
 
   componentDidMount() {
-    getTakenHolidays()
-      .then(takenHolidays => this.setState({ takenHolidays }));
-
-    getRemainingHolidays()
-      .then(remainingHolidays => this.setState({ remainingHolidays }));
-
-    userProfile()
-      .then(employee => this.setState(
-        {
-          employee: {
-            forename: employee.forename,
-            surname: employee.surname,
-          },
-        }
-      ));
+    const { navigation } = this.props;
+
+    this.sub = navigation.addListener('didFocus', () => {
+      getTakenHolidays()
+        .then(takenHolidays => this.setState({ takenHolidays }));
+
+      getRemainingHolidays()
+        .then(remainingHolidays => this.setState({ remainingHolidays }));
+
+      userProfile()
+        .then(employee => this.setState({ employee }));
```

Comment:
Still gonna approve, but it may be good to get some error handling on these calls

Reply:
I do have some error handling set within the service calls ( displays an alert ). Do think it would be better to set them within the container files or keep them within services or utilities folder?

Reply:
@kirsty-unosquare That's a cool idea. I think that depends on if your setup makes it easy or not to add additional logic in specific cases. (For example, navigate away from a screen on an error). Sounds absolutely fine to me for right now though, so again, approving 👍

Reply:
Sweet gonna merge this branch.